models/endpoints.model.js

```diff
@@ -20,7 +20,8 @@ exports.selectAllArticles = (sort_by = 'created_at', order = 'desc') => {
         LEFT JOIN comments ON articles.article_id = comments.article_id
         GROUP BY articles.article_id`;
 
-    queryStr += ` ORDER BY ${sort_by} ${order.toUpperCase()};` 
+    queryStr += format(` ORDER BY %s %s;`, sort_by, order.toUpperCase())
```

Comment:
You can use `%I` for column names. This is an escaped value so is safer than `%s`, you will need `%s` for the order though (or string interpolation)